functions/packages.bash

```diff
@@ -132,15 +133,15 @@ exim_setup() {
 ##    etckeeper_setup()
 ##
 etckeeper_setup() {
-  if ! [ -x "$(command -v etckeeper)" ]; then
+  if ! [[ -x $(command -v etckeeper) ]]; then
     echo -n "$(timestamp) [openHABian] Installing etckeeper (git based /etc backup)... "
     if cond_redirect apt-get install --yes etckeeper; then echo "OK"; else echo "FAILED"; return 1; fi
   fi
 
   echo -n "$(timestamp) [openHABian] Configuring etckeeper (git based /etc backup)... "
   if ! cond_redirect sed -i 's/VCS="bzr"/\#VCS="bzr"/g' /etc/etckeeper/etckeeper.conf; then echo "FAILED"; return 1; fi
   if ! cond_redirect sed -i 's/\#VCS="git"/VCS="git"/g' /etc/etckeeper/etckeeper.conf; then echo "FAILED"; return 1; fi
-  if cond_redirect bash -c "cd /etc && etckeeper init && git config user.email 'etckeeper@localhost' && git config user.name 'openhabian' && git commit -m 'initial checkin' && git gc"; then echo "OK"; else echo "FAILED"; return 1; fi
+  if cond_redirect bash -c "cd /etc && etckeeper init && git config user.email 'etckeeper@openHABian' && git config user.name 'openhabian' && git commit -m 'initial checkin' && git gc"; then echo "OK"; else echo "FAILED"; return 1; fi
```

Comment:
hmm, does either of these email addresses / github accounts make sense ?
Did you ever use this?

Reply:
Truthfully, no I tested it but it is not even a menu option. I am not sure what it was intended for so I left it. @ThomDietrich any hints?

Reply:
I've extended Amanda backup tool config to backup all of /etc (instead of /etc/openhab2 only)